functions.php

```diff
@@ -45,3 +46,46 @@ function pullData(PDO $db):array{
 $players = $query->fetchAll();
 return $players;
 }
+
+
+
+
+/**
+ * This function prepares the stats for entry into the databse by binding them to params and making a connection to the db
+ *
+ * @param string $name
+ * @param int $points
+ * @param int $games
+ * @param int $rings
+ * @param PDO $db
+ */
+function insertData (string $name,int $points,int $games,int $rings,PDO $db){
+    $dbInsert = $db->prepare("INSERT INTO `nba` (`name`, `points`, `games`, `rings`) VALUES (:player, :points, :games, :rings)");
+    if (strlen($name) <= 30){
+        $dbInsert->bindParam(':player', $name);
+        $dbInsert->bindParam(':points', $points);
+        $dbInsert->bindParam(':games', $games);
+        $dbInsert->bindParam(':rings', $rings);
+        $dbInsert->execute();
```

Comment:
execute returns a boolean of if the query was successful, so you can actually return this line instead of manually returning true